src/Integration/Binding/IJsTsProjectTypeIndicator.cs

```diff
@@ -18,14 +18,16 @@
  * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
  */
 
-namespace SonarLint.VisualStudio.Core.JsTs
+using EnvDTE;
+
+namespace SonarLint.VisualStudio.Integration.Binding
 {
     public interface IJsTsProjectTypeIndicator
     {
         /// <summary>
-        /// Returns true if the currently open solution has a JS/TS file.
-        /// Returns false if the solution has no JS/TS files, or if there is no open solution.
+        /// Returns true if the currently open solution/folder has a JS/TS file.
```

Comment:
The word "solution" is misleading here. We're not checking the solution but the given project. There is also no mention of what happens if the project is null and we are in open folder/not in open as folder.